modules/nf-core/atlasgeneannotationmanipulation/gtf2featureannotation/main.nf

```diff
@@ -0,0 +1,43 @@
+process ATLASGENEANNOTATIONMANIPULATION_GTF2FEATUREANNOTATION {
+    tag '$gtf'
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::atlas-gene-annotation-manipulation=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/atlas-gene-annotation-manipulation%3A1.1.0--hdfd78af_0':
+        'quay.io/biocontainers/atlas-gene-annotation-manipulation:1.1.0--hdfd78af_0' }"
+
+    input:
+    val meta
+    path gtf
+    path fasta
+
+    output:
+    tuple val(meta), path("*.anno.tsv")    , emit: feature_annotation
+    tuple val(meta), path("*.fa.gz")                    , emit: filtered_cdna, optional: true
+    tuple val(meta), path("versions.yml")               , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: ( meta.id ?: '' )
```

Comment:
```suggestion
    def prefix = task.ext.prefix ?: meta.id 
```
As far as I know meta map always has an id (either sample or case) so in case the gtf doesn't have that I think we would want to know. Otherwise in this case, we might just end up creating a hidden file by the name ".anno.tsv" if meta.id isn't set 😅

Reply:
Another good point- thanks!